src/main/ngapp/src/app/chat-bar/chat-bar.component.html

```diff
@@ -12,7 +12,7 @@
 
 <nb-card>
     <nb-list>
-        <nb-list-item *ngFor="let user of users">
+        <nb-list-item *ngFor="let user of users" (click)="onSelect(user.userId)">
```

Comment:
not sure, but does clicking user make the background holding the username to change color?

Reply:
I don't think so. The background currently isn't changed at all

Reply:
ok.

Reply:
curious; is there a way to do that here?

Reply:
Yep. Let me push that in this branch